test/lib/Elastica/Test/ClientTest.php

```diff
@@ -26,6 +26,16 @@ public function testConstruct()
     /**
      * @group functional
      */
+    public function testGetVersion()
+    {
+        $client = $this->_getClient();
+        $this->assertNotEmpty($version = $client->getVersion());
+        $this->assertSame($version, $client->getVersion());
```

Comment:
Could we use here some like version compare to compare it >= 2.0.0? http://php.net/manual/en/function.version-compare.php